frontend/src/Search/Table/CategoryLabel.js

```diff
@@ -1,10 +1,22 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import Label from 'Components/Label';
+import { kinds, tooltipPositions } from 'Helpers/Props';
+import Tooltip from '../../Components/Tooltip/Tooltip';
 
-function CategoryLabel({ categories = [] }) {
+function CategoryLabel({ categories }) {
   const sortedCategories = categories.filter((cat) => cat.name !== undefined).sort((c) => c.id);
 
+  if (categories.length === 0) {
```

Comment:
This will error if categories comes in null/undefined right? I think that was the reason for the default prop

Reply:
Fair point, tested your concerns and it was line 8 that broke first due to filter, so I added back in `defaultProps` to resolve that 👌